test_package/CMakeLists.txt

```diff
@@ -6,6 +6,17 @@ conan_set_vs_runtime()
 conan_set_libcxx()
 conan_output_dirs_setup()
 
+find_package(qt REQUIRED CONFIG)
```

Comment:
why are there 2 calls to find_package (L9 and L32) ?

Reply:
I am not a qt expert, so I might be wrong, but here is how I see it.
First `find_package(qt...)` (important *qt*) is so cmake find and load `qtConfig.cmake` which loads `qtTarget-debug.cmake` and `qtTarget-release.cmake` in the same time (so Debug/Release can be change in IDE (VS, XCode) after generation). But it only loads it (in order to use it `target_link_libraries(${PROJECT_NAME} qt::qt)` must be called).
However Qt has MOC, translations etc, so to run/enable them `find_package(Qt5 ... Core)` (important *Qt5* (not "qt")) must be called.

Qt is huge beast and I am definitely not expert here. If I have to guess I would say that recipe has missing `self.cpp_info.name = "<PKG_NAME>"` (or at least `self.cpp_info.names["generator_name"] = "<PKG_NAME>"` can be applied) where `<PKG_NAME>` should be *Qt5* (not qt).  Maybe that reduce one "find_package".
I guess it also enable using Qt4 and Qt5 in one project in the same time (in the future Qt5 and Qt6).

I am avoiding this area since having [VTK](https://github.com/Kitware/VTK) and qt-tools is much more import for me right now, and not even sure if I will find time for it. Without Your help even finishing this wouldn't be possible - Thank You!!